e2e/runner/e2etest.go

```diff
@@ -103,6 +103,9 @@ func (r *E2ERunner) GetE2ETestsToRunByConfig(
 		if !found {
 			return nil, fmt.Errorf("e2e test %s not found", testSpec.Name)
 		}
+		if r.TestFilter != nil && !r.TestFilter.MatchString(e2eTest.Name) {
```

Comment:
Consider using []string for TestFilterer , to run multiple tests.
We can add filter multiple name using a regex but a comma seprated list might be easier to use

Reply:
Actually I think its fine , something like this seems to work 
r.TestFilter = regexp.MustCompile(`^(stress_solana_withdraw|stress_solana_deposit)$`)

Maybe just documenting the usage should be enough.